package.json

```
@@ -1,18 +1,20 @@
 {
   "name": "shadow-study",
   "private": true,
-  "version": "1.0.0",
+  "version": "1.1.0",
```

We don't maintain any version specifier here, so keep this as 1.0.0 (it can really be whatever, as it's not really used for anything.)
```suggestion
  "version": "1.0.0",
```

---

Kept version as 1.0.0.